apps/admin-gui/src/app/admin/pages/admin-page/admin-services/service-detail-page/service-detail-page.component.html

```diff
@@ -45,6 +45,14 @@
             </mat-icon>
           </button>
         </span>
+        <button
+          *ngIf="authResolver.isPerunAdmin()"
+          (click)="evaluateConsents()"
+          color="accent"
+          class="action-button mr-2"
```

Comment:
Set up a margin between this button and the service status. Add a `ml-2` or `ml-3` class.